packages/react-shopify-app-bridge/src/Provider.tsx

```diff
@@ -17,14 +16,12 @@ export enum AppType {
 /** Internal props used to create the right structure of providers */
 type GadgetProviderProps = {
   children: JSX.Element | JSX.Element[];
-  forceRedirect: boolean;
-  isEmbedded: boolean;
   gadgetAppUrl: string;
   originalQueryParams?: URLSearchParams;
   api: AnyClient;
-  isRootFrameRequest: boolean;
-  isInstallRequest: boolean;
   type: AppType;
+  host: string | undefined;
```

Comment:
this change in props  will mean upgrade steps existing apps right?

Reply:
sorry these are just the props for the `InternalGadgetProvider` not the publicly accessible ones. although i did remove the `router` prop on the `GadgetProvider` since the recommendation is now to simply use `history`